pkg/sdn/api/v1/conversion.go

```diff
@@ -2,27 +2,75 @@ package v1
 
 import (
 	kapi "k8s.io/kubernetes/pkg/api"
+	conversion "k8s.io/kubernetes/pkg/conversion"
 
 	oapi "github.com/openshift/origin/pkg/api"
-	"github.com/openshift/origin/pkg/sdn/api"
+	newer "github.com/openshift/origin/pkg/sdn/api"
+)
+
+const (
+	KnownInvalidVNID = ^uint(0)
```

Comment:
+1 to fixed width
-1 to ints > 53 bits as numbers in the API... those will cause serialization/deserialization problems. See:
https://github.com/kubernetes/kubernetes/issues/17095
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#primitive-types